easybuild/framework/easyblock.py

```diff
@@ -1019,7 +1019,18 @@ def make_module_extend_modpath(self):
             # module path extensions must exist, otherwise loading this module file will fail
             for modpath_extension in full_path_modpath_extensions:
                 mkdir(modpath_extension, parents=True)
-            txt = self.module_generator.use(full_path_modpath_extensions)
+            quoted_modpath_exts = [quote_str(ext) for ext in full_path_modpath_extensions]
+            txt = self.module_generator.use(quoted_modpath_exts)
+
+            # add user-specific module path; use statement will be guarded so no need to create the directories
+            user_modpath = build_option('subdir_user_modules')
+            if user_modpath:
+                user_modpath_exts = ActiveMNS().det_user_modpath_extensions(self.cfg)
+                self.log.debug("Including user module path extensions returned by naming scheme: %s"
+                               % user_modpath_exts)
+                quoted_user_modpath_exts = [quote_str(os.path.join(user_modpath, ext)) for ext in user_modpath_exts]
```

Comment:
maybe we should be quoting in `module_generator.use` instead, if it's so important to do this?